apollo-runtime/src/main/java/com/apollographql/apollo/internal/subscription/RealSubscriptionManager.java

```diff
@@ -165,13 +170,16 @@ void doSubscribe(Subscription subscription, SubscriptionManager.Callback callbac
         timer.cancelTask(INACTIVITY_TIMEOUT_TIMER_TASK_ID);
 
         final UUID subscriptionId = UUID.randomUUID();
-
         subscriptions.put(subscriptionId, new SubscriptionRecord(subscriptionId, subscription, callback));
+
         if (state == SubscriptionManagerState.DISCONNECTED) {
           state = SubscriptionManagerState.CONNECTING;
           transport.connect();
         } else if (state == SubscriptionManagerState.ACTIVE) {
-          transport.send(new OperationClientMessage.Start(subscriptionId.toString(), subscription, scalarTypeAdapters));
+          transport.send(
+              new OperationClientMessage.Start(subscriptionId.toString(), subscription, scalarTypeAdapters, autoPersistSubscription,
+                  false)
```

Comment:
This looks little weird in terms of code style. Would be nice to leave this false alone